python/lsst/sconsUtils/dependencies.py

```diff
@@ -504,7 +504,6 @@ def _tryImport(self, name):
                 else:
                     module.config.addCustomTests(self.customTests)
                 return module
-        state.log.info("Failed to import configuration for optional package '%s'." % name)
```

Comment:
So this message was triggering if none of the paths existed (or there were no paths at all)? This message was introduced by @TallJimbo in commit 0c7d930d598811d0862c65a666dd9a89be23250d back in 2011 and has moved around the code base a bit since then being tweaked along the way.